proxy/src/main/java/com/wavefront/agent/PostSourceTagTimedTask.java

```diff
@@ -145,6 +159,8 @@ public void run() {
       }
     } catch (Throwable t) {
       logger.log(Level.SEVERE, "Unexpected error in flush loop", t);
+    } finally {
+      scheduler.schedule(this, nextRunMillis, TimeUnit.MILLISECONDS);
```

Comment:
Just a comment, this doesn't enforce running this every nextRunMillis. Also I am not sure why you need the variable nextRunMillis